packages/block-editor/src/components/editor-styles/index.js

```diff
@@ -66,7 +66,56 @@ function useDarkThemeBodyClassName( styles ) {
 	);
 }
 
-export default function EditorStyles( { styles } ) {
+function useParsedAssets( html ) {
+	return useMemo( () => {
+		const doc = document.implementation.createHTMLDocument( '' );
+		doc.body.innerHTML = html;
+		return Array.from( doc.body.children );
+	}, [ html ] );
+}
+
+function PrefixedStyle( { tagName, href, id, rel, media, textContent } ) {
+	const TagName = tagName.toLowerCase();
+
+	function onLoad( event ) {
+		const { sheet } = event.target;
+		const { cssRules } = sheet;
+
+		let index = 0;
+
+		for ( const rule of cssRules ) {
+			const { type, STYLE_RULE, selectorText, cssText } = rule;
+
+			if ( type !== STYLE_RULE ) {
+				continue;
```

Comment:
`CSSRule.type` [is deprecated](https://developer.mozilla.org/en-US/docs/Web/API/CSSRule/type). Looking at the [documentation](https://developer.mozilla.org/en-US/docs/Web/API/CSSRule) and testing locally, I think the correct approach is to look at the instance of the rule:

```suggestion
			if ( ! ( rule instanceof CSSStyleRule ) ) {
				continue;
```